src/components/Main/Results/Utils.test.ts

```diff
@@ -35,3 +35,40 @@ describe('verifyTuple', () => {
     expect(verifyTuple([1, 2])).toStrictEqual([1, 2])
   })
 })
+
+describe('computeNewEmpiricalCases', () => {
+  /* Note that this test presumes the 'time' field is not used by the computeNewEmpiricalCases */
+  /* The assumption is correct at the time of writing, and exploited to make the test simpler. */
+  /* But the time field must be present to keep the typescript compiler happy. */
+  const time = new Date(0)
+  const cumulativeCounts = [
+    { cases: 1, time },
+    { cases: 2, time },
+    { cases: 4, time },
+    { cases: 6, time },
+    { cases: 6, time },
+    { cases: 7, time },
+  ]
+  it('returns empty result if cumulativeCounts parameter is undefined.', () => {
+    const got = computeNewEmpiricalCases(3)
+    expect(got).toStrictEqual([[], 3])
+  })
+
+  it('computes with a timeWindow of 1', () => {
+    const [newEmpiricalCases, deltaDay] = computeNewEmpiricalCases(1, cumulativeCounts)
+    expect(newEmpiricalCases).toEqual([undefined, 1, 2, 2, undefined, 1])
+    expect(deltaDay).toEqual(1)
+  })
+
+  it('computes with a timeWindow that evenly divides cumulativeCounts', () => {
+    const [newEmpiricalCases, deltaDay] = computeNewEmpiricalCases(2, cumulativeCounts)
+    expect(newEmpiricalCases).toEqual([undefined, undefined, 3, 4, 2, 1])
+    expect(deltaDay).toEqual(2)
+  })
+
+  it('computes with a timeWindow that does not evenly divide cumulativeCounts', () => {
+    const [newEmpiricalCases, deltaDay] = computeNewEmpiricalCases(4, cumulativeCounts)
+    expect(newEmpiricalCases).toEqual([undefined, undefined, undefined, undefined, 5, 5])
+    expect(deltaDay).toEqual(4)
+  })
+})
```

Comment:
Tests for the `computeNewEmpiricalCases()` function. As noted in the comment, the `time` property is not used by the function but is required by the type. Therefore it needs to be present but its value is of no consequence.